v2/cf-nodejs-logging-support/src/lib/logger/logger.ts

```diff
@@ -39,10 +40,16 @@ export default class Logger {
 
     logMessage(levelName: string, ..._args: any) {
         if (!this.isLoggingLevel(levelName)) return;
-        const record = this.context ? RecordFactory.buildMsgRecord(_args, this.context) : RecordFactory.buildMsgRecord(_args);
+        const record = this.context ?
+            RecordFactory.buildMsgRecord(this.registeredCustomFields, this.customFields, levelName, _args, this.context)
+            : RecordFactory.buildMsgRecord(this.registeredCustomFields, this.customFields, levelName, _args);
```

Comment:
Is this check necessary? From my understanding `this.context` gets checked again in `buildMsgRecord`.